src/sentry/flags/endpoints/secrets.py

```diff
@@ -29,24 +29,18 @@ class FlagWebhookSigningSecretResponse(TypedDict):
 @register(FlagWebHookSigningSecretModel)
 class FlagWebhookSigningSecretSerializer(Serializer):
     def serialize(self, obj, attrs, user, **kwargs) -> FlagWebhookSigningSecretResponse:
-        secret = obj.secret
-        if len(secret) >= 10:
-            secret = secret[0:6] + "*" * len(secret[6:])
-        else:
-            secret = "*" * len(secret)
-
         return {
             "createdAt": obj.date_added.isoformat(),
             "createdBy": obj.created_by,
             "id": obj.id,
             "provider": obj.provider,
-            "secret": secret,
+            "secret": obj.secret[0:6] + "*" * (len(obj.secret) - 6),
```

Comment:
I think this will error if len < 6, multiplying string by negative number

Reply:
![Uploading Screenshot 2024-12-03 at 10.24.31 AM.png…]()
Ok nvm, python continues to impress 😂

Reply:
Its validated to be length 32.

Reply:
Oh nvm just saw your other comment :D